package/yast2-users.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Tue Jul 20 15:09:28 UTC 2021 - David Diaz <dgonzalez@suse.com>
+
+- Improve handling of authorized_keys file for recreating it
```

Comment:
I would say "Do not rewrite authorized_keys unless it is needed.". Or even "Do not modify authorized_keys...".